resources/util.go

```diff
@@ -133,3 +135,35 @@ func LowerStructFieldNameToFieldName(res Res) (map[string]string, error) {
 	}
 	return result, nil
 }
+
+// GetUID returns the UID of an user
+// Caller should first check user is not empty
+func GetUID(username string) (int, error) {
+	u2, err2 := user.LookupId(username)
+	if err2 == nil {
+		return strconv.Atoi(u2.Uid)
+	}
+
+	u, err := user.Lookup(username)
+	if err == nil {
+		return strconv.Atoi(u.Uid)
+	}
+
+	return -1, errwrap.Wrapf(err, "owner lookup error (%s)", username)
+}
+
+// GetGID returns the GID of a group
+// Caller should first check group is not empty
+func GetGID(group string) (int, error) {
+	g2, err2 := user.LookupGroupId(group)
```

Comment:
rename similarly to other review comment above please.

Reply:
Done as part of https://github.com/purpleidea/mgmt/pull/256#discussion_r152585679